passout-scenes ~추위/더위 기절 겸 질문

extraslugabed opened the pull request:
542번줄 <<= _clothes.join(" and ")>>에 조사를 어떻게 넣는지 모르겠네용..

extraslugabed committed:
~추위/더위 기절

extraslugabed committed:
passout-scenes.twee ~rescue

542번 줄 <<= _clothes.join(" and ")>>에 어떻게 조사를 붙여야 하는지 모르겠네요.. 남겨둠

extraslugabed committed:
Update passout-scenes.twee

extraslugabed committed:
Update passout-scenes.twee

uotalkie commented:
@extraslugabed 
답변이 늦어서 죄송합니다. _clothes 변수는 hypothermiaClothes 위젯에서 설정되어 나오는 것으로 아직 번역이 되어있지 않습니다.  그대로 두시면 나중에 merge 할 때 해당 위젯을 포함해서 번역을 하도록 하겠습니다.

extraslugabed committed:
Update passout-scenes.twee

extraslugabed committed:
Update passout-scenes.twee

extraslugabed committed:
Update passout-scenes.twee

<<= $passoutReason>>, grand<<father>>등
어떻게 처리하면 될지 모르겠는 (아마 온도 시스템 파트가 한패에 추가된지 얼마 안 돼서 그런걸까 하는)부분들은 일단 그대로 두었어요
외에는 번역 끝(오타 수정도)
이거 정말 수고 많이 들어가는 일이구나 고생많으십니다 감사합니다
